blocks/global-phrases-block/intl.json

```diff
@@ -130,5 +130,38 @@
 		"no": "(Åpnes i et nytt vindu)",
 		"pt": "(Abre numa nova janela)",
 		"sv": "(Öppnas i nytt fönster)"
+	},
+	"global.byline-by-text": {
```

Comment:
Can we make sure we update the locale files in the `locale` folder in the root of the repo for these

Reply:
done